.github/workflows/linux.yml

```diff
@@ -327,15 +327,15 @@ jobs:
     uses: ./.github/workflows/job_openvino_js.yml
     with:
       runner: 'aks-linux-4-cores-16gb'
-      container: '{"image": "${{ fromJSON(needs.docker.outputs.images).ov_build.ubuntu_20_04_x64 }}"}'
+      container: '{"image": "${{ fromJSON(needs.docker.outputs.images).ov_build.ubuntu_22_04_x64 }}"}'
 
   Conformance:
     needs: [ Build, Smart_CI ]
     timeout-minutes: ${{ matrix.TEST_TYPE == 'API' && 5 || 20 }}
     defaults:
       run:
         shell: bash
-    runs-on: ubuntu-20.04-8-cores
+    runs-on: ubuntu-latest-16-cores
```

Comment:
The `ubuntu-latest-8-cores` runner repeatedly failed with `[error]The runner has received a shutdown signal. This can happen when the runner service is stopped, or a manually started runner is canceled.`, an [example](https://github.com/openvinotoolkit/openvino/actions/runs/9414032835/job/25932847579).

It seems like a known unresolved bug with the `latest` Ubuntus. [1](https://github.com/actions/runner-images/issues/6709) + [2](https://github.com/actions/runner-images/discussions/7188). The suggestions are:
* to use a bigger runner - seems to be working: https://github.com/openvinotoolkit/openvino/actions/runs/9414801191
* use swap - TBD